config/opal_check_xpmem.m4

```diff
@@ -103,7 +103,11 @@ AC_DEFUN([OPAL_CHECK_BASE_XPMEM], [
                       [opal_check_xpmem_base_happy="no"])
 
      AS_IF([test "${opal_check_xpmem_base_happy}" = "yes"],
-           [$2],
+           [$1_LDFLAGS="[$]$1_LDFLAGS $XPMEM_LIBS"
```

Comment:
You don't need to set $1_LDFLAGS, CPPFLAGS, or LIBS.  OAC_CHECK_PACKAGE will do that for you.  You do need the AC_DEFINE_UNQUOTED, of course.

Reply:
Thanks for the feedback!